Cosmos DB Emulator 1.20.108.4

David-Noble-at-work opened the pull request:
This change updates the change list for the February 14, 2018 release of the Cosmos DB Emulator.

azurecla commented:
Hi __@dtzar__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

ggailey777 commented:
@dtzar This PR has been open for a while and there is now a conflict. If you still want this submission merged, please resolve the conflict. Otherwise, I can close it for you.

ggailey777 commented:
Sorry we weren't able to use your contribution this time. Please don't hesitate to submit improvements to Azure docs in the future.

Commit:
Cosmos DB Emulator 1.20.108.4

This change updates the change list for the February 14, 2018 release of the Cosmos DB Emulator.

Commit:
Tweaked version number on prior release for greater specificity
